docusaurus/docs/migration-guide/v10.0.x-v10.1.x/index.md

```diff
@@ -0,0 +1,22 @@
+---
+id: migrate-10.0.x-to-10.1.x
+title: 10.0.x to 10.1.x
```

Comment:
Docusaurus has both keywords and tags, and I haven't had the time yet to analyze which of these are preferable and how they appear in the UX. The tags in Hugo were basically worthless, as they were not displayed but only used to help the website display related topics, and Docusaurus site doesn't have a related topics feature. So what would they be used for? We might need advice from our SEO expert to make a decision.

Regarding descriptions, I haven't found anything in Docusaurus's documentation with this name. Have you? I imagine that if there is such a tag then it would probably have one function: to supply the excerpt content for use in the DocCardList, which we aren't using. If there's a use for this metadata though, let's add it back in.

Reply:
> Regarding descriptions, I haven't found anything in Docusaurus's documentation with this name.
The table on [this page](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#description) explains all the fields that can be used as metadata / front matter. 

> The tags in Hugo were basically worthless, as they were not displayed but only used to help the website display related topics, and Docusaurus site doesn't have a related topics feature. So what would they be used for?
Both keywords and description are used by docusaurus to render html for SEO.

> We might need advice from our SEO expert to make a decision.
Probably worth getting them involved. I think the description meta tags will be needed as they also exist in the hugo built html.

Reply:
I see what you mean. The description default is the first sentence, which is usually what we want it to be, I think. But we should still add these, at least when we want the description to be different. Tags we can add for now as well, but I do want to check with SEO expert to get their recommendations. I've updated them for this PR.